pom.xml

```diff
@@ -207,6 +206,26 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>software.amazon.kinesis</groupId>
+            <artifactId>amazon-kinesis-client</artifactId>
+            <version>2.2.9</version>
```

Comment:
Should we also add these dependencies with scope `provided`? (To avoid that they get packaged into the localstack-utils JAR file that we publish on Maven Central.)